SDK_dotnet/Utils/GluwacoinConverter.cs

```diff
@@ -10,10 +10,17 @@ public static BigInteger ConvertToGluwacoinBigInteger(string amount)
             BigDecimal bigDecimalAmount = BigDecimal.Parse(amount);
             return BigInteger.Parse((bigDecimalAmount * new BigDecimal(1, 18)).Floor().ToString());
         }
+
         public static BigInteger ConvertToGluwacoinSideChainBigInteger(string amount)
```

Comment:
@kyeongminkang Consider making this a single method with a default number of decimals that can be overridden if needed

```
        public static BigInteger ConvertToGluwacoinSideChainBigInteger(string amount, int decimals = 18)
        {
            BigDecimal bigDecimalAmount = BigDecimal.Parse(amount);
            return BigInteger.Parse((bigDecimalAmount * new BigDecimal(1, decimals)).Floor().ToString());
        }
```

For sUSDCG you would use:
`ConvertToGluwacoinSideChainBigInteger(amount, 6)`

For sNGNG you would use:
`ConvertToGluwacoinSideChainBigInteger(amount)`